server/src/models/User/index.model.ts

```diff
@@ -51,6 +51,11 @@ export default class User extends Model {
   })
     email: string;
 
+  @Column({
+    type: DataType.BOOLEAN,
+  })
+    emailVerified: boolean;
```

Comment:
maybe change to isEmailVerified 
this could be mistaken with verifiedBy or verifiedAt